core/arch/arm/mm/core_mmu_lpae.c

```diff
@@ -961,10 +1133,13 @@ bool core_mmu_user_mapping_is_active(void)
 void core_mmu_get_user_map(struct core_mmu_user_map *map)
 {
 	struct mmu_partition *prtn = get_prtn();
+	uint64_t *entry = NULL;
 
 	assert(user_va_idx != -1);
```

Comment:
Can remove this test since needed only from `core_mmu_get_user_mapping_entry()` and already asserted there.